config/webpack.config.js

```diff
@@ -52,11 +50,16 @@ var config = {
       path.join(__dirname, '..', 'webpack'),
       'node_modules/'
     ],
-    alias: {
+    alias: Object.assign({
       foremanReact:
         path.join(__dirname,
-           '../webpack/assets/javascripts/react_app/components')
-    }
+           '../webpack/assets/javascripts/react_app'),
+      foremanNodeModules:
+        path.join(__dirname,
+          '../node_modules')
```

Comment:
:+1: this solves an issue of duplicating code in the bundle! but /cc @dLobatog since he introduced this change in the first place.

if you want to visualize the before / after have a look at https://chrisbateman.github.io/webpack-visualizer/